src/main/blackbox/blackbox.c

```diff
@@ -1706,19 +1709,29 @@ void blackboxUpdate(timeUs_t currentTimeUs)
 }
 
 /*
- * Returns start time in ISO 8601 format, YYYY-MM-DDThh:mm:ss
- * Year value of "0000" indicates time not set
+ * Formats start time in ISO 8601 format, YYYY-MM-DDThh:mm:ss
+ * Year value of "0000" indicates time not set.
  */
-static char startDateTime[20] = "0000-01-01T00:00:00";
-const char *blackboxGetStartDateTime(void)
+
+static char started_buf[FORMATTED_DATE_TIME_BUFSIZE];
```

Comment:
@digitalentity I know, it was just because of the macro which calls into `blackboxGetStartDateTime()` and because this function was also exposed in the header. I didn't want to touch the macro nor the public function without some feedback first. Should I just remove `blackboxGetStartDateTime()` from the header?

Reply:
Yes, remove both `blackboxGetStartDateTime` and `blackboxSetStartDateTime`. Use a local buffer and RTC API to get the date/time in `blackboxWriteSysinfo`